### PR TITLE
[extension/jaegerremotesampling] mark as alpha

### LINKED ISSUE #13005: [extension/jaegerremotesampling] Mark as alpha
This component is new and hasn't had exposure to production workloads yet. As such, it should be classified as alpha instead of beta.

### PR DESCRIPTION
**Pull request by frzifus**
Signed-off-by: Benedikt Bongartz <bongartz@klimlive.de>

**Description:** <Describe what has changed.>

**Link to tracking Issue:** <Issue number if applicable>
Closes  #13005

**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

---

@jpkrohling Is that all we have to do or is there something else I haven't considered?

### CODE REVIEW

**Review by jpkrohling — APPROVED** [review]

File: extension/jaegerremotesampling/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 | Status                   |                       |
 | ------------------------ |-----------------------|
-| Stability                | [beta]                |
+| Stability                | [alpha]                |
```
Comment (jpkrohling) [inline extension/jaegerremotesampling/README.md:5]: nit: the table alignment could be better here, now that "alpha" (5 chars) replaced "beta' (4 chars).

**Review by jpkrohling — APPROVED** [review]

File: extension/jaegerremotesampling/README.md (hunk 1 above)
Comment (jpkrohling) [reply extension/jaegerremotesampling/README.md:5]: if you don't want to fix it, let me know and I'll merge as is.

Comment (frzifus) [reply extension/jaegerremotesampling/README.md]: fixed

**Review by jpkrohling — APPROVED** [review]

**Comment by jpkrohling** [comment]
Merging it anyway, as it has passed before removing the single whitespace from the readme. I don't think it's related to that change.

**Pull request merged**